OIPS/oip-draft-versioning.md

```diff
@@ -0,0 +1,41 @@
+---
+oip: 4
+title: Releases Versioning Specification
+author:  Francesco Pacella <f@ost.com>
+discussions-to: -
+status: Draft
+type: Meta
+category -
+created: 2018-11-23
+requires : -
+replaces : -
+---
+
+#Releases Versioning Specification
+
+##Simple Summary
+We need to establish a new consistent versioning standard for OpenST releases going forward and agree on the versioning logic for Mosaic and BrandedToken and other repositories releases as well.
+
+##Abstract
+We started in November 2017 with an initial unique release stream, being OpenST. The first version was released under the name of 0.9.0. The versions that followed did not followed the common standard for releases, utilizing PATCH level to actually address MINOR level releases. 
```

Comment:
Line breaks between new lines of simple text are not rendered in MD output—if the line breaks in this abstract section are meant to place each sentence into its own paragraph, empty lines should be added between.